src/tools/ScenarioMeasurement/Startup/Startup.cs

```diff
@@ -116,7 +120,15 @@ static void checkArg(string arg, string name)
             if (!failed)
             {
                 logger.Log("Parsing..");
-                TraceEventSession.Merge(new[] { kernelTraceFile, userTraceFile }, traceFileName);
+                var files = new System.Collections.Generic.List<string>
```

Comment:
nit: add the using at the top of the file instead of using the full typename here. This follows our general conventions.